test/fixtures/businesses/hiring_invoice_requests.yml

```diff
@@ -0,0 +1,9 @@
+one:
+  business_id: <%= ActiveRecord::FixtureSet.identify(:one) %>
```

Comment:
Was unable to get `business: businesses(:one)` or `business_id: businesses(:one).id`  to work here.

Realized `null:false` on the migration affects this.

Thinking `business_id: <%= ActiveRecord::FixtureSet.identify(:one) %>` is the way to go here.